src/SFA.DAS.ASK.Web/Views/RequestSupport/CheckYourDetails.cshtml

```diff
@@ -74,9 +70,7 @@ else
                         </strong>
                     </dd>
                     <dd class="govuk-summary-list__actions">
-                        @* <a class="govuk-link" href="#"> *@
-                        @*     Change<span class="govuk-visually-hidden"> organisation name</span> *@
-                        @* </a> *@
+                        <a asp-action="Index" asp-controller="OrganisationResults" asp-route-requestId="@Model.RequestId" asp-route-edit="true" class="govuk-link">Change</a>
```

Comment:
We're going to need a switch on this as if this is a dfesignin route, then.... I'm not sure what should happen. 😆